projects/ui/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
huh; I never knew this should be lowercase

Reply:
I think this was actually a prettier fix, and not in the lighthouse score.

Reply:
Apparently it's not case sensitive, so it should be okay either way.